Core/clim-basic/decls.lisp

```diff
@@ -1201,11 +1203,6 @@ rendered on MEDIUM with the style LINE-STYLE."))
 
 ;;;
 
-(defgeneric text-style-character-width (text-style medium char))
-;; fall back, where to put this?
-(defmethod text-style-character-width (text-style medium char)
-  (text-size medium char :text-style text-style))
-
```

Comment:
I'm not sure this change is an improvement. I usually consider unspecialized default methods as part of the protocol. I don't have a super strong opinion on this, though.

Reply:
fixed (I've put it in :method clause of defgeneric to signify the fact that it is part of this protocol function). n.b it is an internal protocol which is expected to be implemented by backends for purpose of extended output stream width specifications (i.e there is no clim:text-style-character-width).